ocrd_utils/ocrd_utils/__init__.py

```diff
@@ -5,29 +5,48 @@
 
 The functions have the syntax X_from_Y, where X/Y can be
 
-points a string encoding a polygon: "0,0 100,0 100,100, 0,100"
-polygon an array of x-y-tuples of a polygon: [[0,0], [100,0], [100,100], [0,100]]
-xywh a dict with keys for x, y, width and height: {'x': 0, 'y': 0, 'w': 100, 'h': 100}
-points is for PAGE
+* bbox is a 4-tuple of x0, y0, x1, y1 of the bounding box
+* points a string encoding a polygon: "0,0 100,0 100,100, 0,100". points is for PAGE
+* polygon an array of x-y-tuples of a polygon: [[0,0], [100,0], [100,100], [0,100]]
+* xywh a dict with keys for x, y, width and height: {'x': 0, 'y': 0, 'w': 100, 'h': 100}
 
 polygon is what opencv2 expects
 
 xywh is what tesserocr expects/produces.
 """
```

Comment:
Please mention the remaining functions in the top-level docstring.
```suggestion
* is_local_filename, safe_filename, abspath

* is_string, membername, concat_padded

* MIMETYPE_PAGE, EXT_TO_MIME, VERSION
"""
```